verification/VerifyMetrics.wdl

```diff
@@ -18,8 +18,8 @@ workflow VerifyMetrics {
     call CompareMetricFiles {
       input:
         dependency_input = CompareNumberOfMetricFiles.output_file,
-        file1 = test_metrics[idx],
-        file2 = truth_metrics[idx],
+        file1 = truth_metrics[idx],
```

Comment:
Is there a reason this was changed?
I don't see what this change does, it appears to do nothing... am I wrong?

Reply:
I wanted to have the expected first so that the reported (signed) deltas make more sense.